lib/js-file.js

```diff
@@ -134,6 +136,50 @@ JsFile.prototype = {
     },
 
     /**
+     * Sets whitespace before specified token.
+     *
+     * @param {Object} token
+     * @param {String} whitespace
+     */
+    setWhitespaceBefore: function(token, whitespace) {
```

Comment:
I guess it is fine for temporary measure, but we might need to start about limit for cyclomatic complexity